pkg/bootstrap/docker/openshift/helper.go

```diff
@@ -686,8 +718,17 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 	if err != nil {
 		return err
 	}
-	nodeCfg.DNSBindAddress = ""
-	nodeCfg.DNSIP = ""
+	version, err := h.serverVersion()
+	if err != nil {
+		return err
+	}
+	if useBindAddress(version) {
+		nodeCfg.DNSBindAddress = "127.0.0.1:53"
+		nodeCfg.DNSIP = "172.30.0.1"
```

Comment:
So just tried... if I leave the bind address blank and only set the dns ip, then I get the right behavior:

```
oc rsh simple-ruby-1-48pcf cat /etc/resolv.conf
search default.svc.cluster.local svc.cluster.local cluster.local
nameserver 172.30.0.1
nameserver 192.168.65.1
options ndots:5
```

I'll change the code to not set the bind address.